src/libostree/ostree-fetcher-soup.c

```diff
@@ -990,14 +941,24 @@ on_stream_read (GObject        *object,
         }
       else
         {
-          g_task_return_pointer (task,
-                                 g_strdup (pending->out_tmpfile),
-                                 (GDestroyNotify) g_free);
+          g_autofree char *uristring
+            = soup_uri_to_string (soup_request_get_uri (pending->request), FALSE);
```

Comment:
Minor: I tend to prefer keeping the `=` on the first line.

Reply:
Yep me too, thanks for catching that.